pkg/config/config.go

```diff
@@ -478,6 +478,13 @@ func (c *Config) UpdateFromFile(path string) error {
 		delete(c.Runtimes, defaultRuntime)
 	}
 
+	// Registries are deprecated in cri-o.conf and turned into a NOP.
+	// Users should use registries.conf instead, so let's log it.
+	if len(t.Crio.Image.Registries) > 0 {
+		t.Crio.Image.Registries = nil
+		logrus.Warnf("The 'registries'option has been deprecated but is referenced in %q.  Please use containers-registries.conf(5) instead.", path)
```

Comment:
this part can go in the 1.20 version, and we can just drop this entirely

Reply:
I see wisdom in keeping this for a longer time.  Not all users go through all versions, docs can be outdated etc.  I think it doesn't hurt us to keep it around but can be a time safer for users.

Reply:
:shrug: it's up to you, because any issues with the field will likely cause us to call you to help :laughing:

Reply:
to me, the messaging is clearer if we drop all cri-o specific fields at once

Reply:
sorry, for some reason I thought there was an additional field we were dropping in this PR...

Reply:
I'll rephrase to state that field does not exist any more.  That may play better with planned deprecation notice for `1.20`.

Reply:
given that, I think this is a good strategy for telling users about the drop.
Given that, I suggest this wording: 
`Support for the 'registries' field has been dropped but is referenced in %q...`

Reply:
ignore me, this is good, besides the wording nit

Reply:
SGTM :)